.github/workflows/test.yaml

```diff
@@ -25,6 +25,15 @@ jobs:
       - name: Prepare local.json
         run: |
           echo '{
+            "axelar": {
+              "chainId": "axelar-local-1",
```

Comment:
why is this needed if you're overriding the domain separator via CLI and not computing it?